adalfetchclient: Type definition is not found

### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [1.0.3]

### Expected / Desired Behavior / Question

Typings reference for `types.d.ts` is not found. When transpiling TS to JS I get the following error: 

```node_modules/@pnp/nodejs/src/net/adalfetchclient.d.ts(1,22): error TS6053: File '/Users/user/project/packages/nodejs/src/net/types.d.ts' not found.```

<img width="764" alt="screenshot 2018-03-12 19 38 10" src="https://user-images.githubusercontent.com/2900833/37302833-ed9d1a64-262c-11e8-8337-6fe5c597a05a.png">

--- a/packages/nodejs/src/net/adalfetchclient.ts
+++ b/packages/nodejs/src/net/adalfetchclient.ts
@@ -1,28 +1,27 @@
-/// <reference path="types.d.ts" />
 declare var global: any;
 declare var require: (path: string) => any;
 import { AuthenticationContext } from "adal-node";
 const nodeFetch = require("node-fetch");
 import { Util, HttpClientImpl } from "@pnp/common";
 
-interface IAuthenticationContext {
+// interface IAuthenticationContext {
 
-    new(authorityUrl: string): IAuthenticationContext;
-    acquireTokenWithClientCredentials(resource: string, clientId: string, clientSecret: string, callback: (err: any, token: AADToken) => void): void;
-}
+//     new(authorityUrl: string): IAuthenticationContext;
+//     acquireTokenWithClientCredentials(resource: string, clientId: string, clientSecret: string, callback: (err: any, token: any) => void): void;
+// }
 
 export interface AADToken {
     accessToken: string;
     expiresIn: number;
-    expiresOn: Date;
-    isMRRT: boolean;
+    expiresOn: string | Date;
+    isMRRT?: boolean;
     resource: string;
     tokenType: string;
 }
 
 export class AdalFetchClient implements HttpClientImpl {
 
-    private authContext: IAuthenticationContext;
+    private authContext: any;
 
     constructor(private _tenant: string,
         private _clientId: string,

--- a/packages/nodejs/src/net/types.d.ts
+++ b/packages/nodejs/src/net/types.d.ts
@@ -1,2 +1,0 @@
-declare module "adal-node";
-declare module "node-fetch";

